File: Python/Sibice.py
import math

def main():
    list = input().split()
    n = int(list[0])
    l = int(list[1])
    w = int(list[2])
    max = math.sqrt(l * l + w * w)
    for i in range(0,n):
        q = int(input())
        if q <= max:
            print("DA")
        else:
            print("NE")
main()

File: C++/Sibice.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {
  int n;
  double l, w;
  cin >> n >> l >> w;
  double max = sqrt(l * l + w * w);
  for (int i = 0; i < n; i++) {
    int size;
    cin >> size;
    if (size <= max) {
      cout << "DA" << endl;
    } else {
      cout << "NE" << endl;
    }
  }
}
